apps/web/src/components/organization-select.tsx

```diff
@@ -42,16 +42,18 @@ import { Input } from "./ui/input";
 type OrganizationForm = Omit<InsertOrganization, "owner">;
 
 export default function OrganizationSelect() {
-  const organizationFromPathname = decodeURIComponent(
-    // @ts-expect-error Since we are taking something from a pathname there has to be something
-    usePathname().split("/").at(2),
-  );
-
```

Comment:
Why you deleted the decoding? Wouldn't it reintroduce a bug with special characters from the URL?

Reply:
It won't since there cannot be any special characters in the name right now.